docker-rollout

```diff
@@ -72,84 +74,119 @@ scale() {
 }
 
 main() {
-  OLD_CONTAINER_ID=$(docker ps --filter "label=com.docker.compose.service=$SERVICE" --format "{{.ID}}")
-
-  if [[ "$OLD_CONTAINER_ID" == "" ]]; then
-    echo "=> Service '$SERVICE' is not running. Starting the service."
-    # COMPOSE_FILES must be unquoted to allow multiple files
-    # shellcheck disable=SC2086
-    $COMPOSE_COMMAND $COMPOSE_FILES up --detach "$SERVICE"
+  # shellcheck disable=SC2086
+  if ! ($COMPOSE_COMMAND $COMPOSE_FILES ps -q "$SERVICE"); then
+    echo "==> Service '$SERVICE' is not running. Starting the service."
+    $COMPOSE_COMMAND $COMPOSE_FILES up --detach --scale="$SERVICE=$SCALE" --no-recreate "$SERVICE"
     exit 0
   fi
 
-  if [[ $(echo "$OLD_CONTAINER_ID" | wc -l) -gt 1 ]]; then
-    echo "Service '$SERVICE' has more than one container running. Make sure 'scale' is set to 1." >&2
-    echo "Fix with: $COMPOSE_COMMAND up --detach --scale $SERVICE=1 $SERVICE" >&2
-    exit 1
+  # shellcheck disable=SC2086
+  if [[ "$($COMPOSE_COMMAND $COMPOSE_FILES ps -q "$SERVICE")" == "" ]]; then
+    echo "==> Service '$SERVICE' is not running. Starting the service."
+    $COMPOSE_COMMAND $COMPOSE_FILES up --detach --scale="$SERVICE=$SCALE" --no-recreate "$SERVICE"
+    exit 0
   fi
```

Comment:
Was it intended to check `! ($COMPOSE_COMMAND $COMPOSE_FILES ps -q "$SERVICE")` and then `"$($COMPOSE_COMMAND $COMPOSE_FILES ps -q "$SERVICE")" == ""` separately? Isn't a single `if` sufficient in this case?